Spanner/src/Connection/ConnectionInterface.php

```diff
@@ -103,6 +103,11 @@ public function updateBackup(array $args);
      */
     public function createBackup(array $args);
 
+    /**
+     * @param array $args
+     */
+    public function copyBackup(array $args);
```

Comment:
This technically breaks BC because anyone implementing this interface would get a fatal error when they update. This _might_ be OK since these interfaces are more-or-less internal, but they are not marked as such, so we should find a precedent first.

Reply:
Hey @bshaffer 
Can I proceed with this or should I remove this from the interface and add it back in the next major version.
This doesn't seem right as the method definitely needs to exist.

Reply:
I defer to @dwsupplee on this.i believe it's ok because there is no way for customers to provide these connection classes AFAIK.

Reply:
@dwsupplee has said we have made changes like this in the past to these classes so your change here should be okay 😄